.github/workflows/publish.yml

```diff
@@ -27,6 +33,7 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           context: .
+          platforms: linux/amd64,linux/arm64
```

Comment:
what we need is all of these platforms build:
- linux/amd64
- linux/arm64
- linux/arm/v7
- linux/arm64/v8

Right now I'm improving the docs and Dockerfile to allow for `arm/v7` as it's failing without a qemu binary in the container